python/test/unit/operators/test_blocksparse.py

```diff
@@ -3,15 +3,12 @@
 import pytest
 
 
-@pytest.mark.parametrize(
-    "MODE, TRANS_A, TRANS_B, BLOCK, DTYPE",
-    [
-        (mode, at, bt, block, dtype) for dtype in ["float16"] for mode in ["sdd", "dsd", "dds"]
-        for at in [False, True] for bt in [False, True] for block in [16, 32, 64]
-    ],
-)
+@pytest.mark.parametrize("MODE", ["sdd", "dds", "dsd"])
```

Comment:
taste and colours, I feel like this makes it a little more explicit what values are covered for a given setting. pytest supports stacking up @parametrize just fine